### PR TITLE
TELCODOCS-1671 adding missed bug fixes missed in z-stream

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1671]: Adding note clarifying when bug is fixed and in what z stream

Version(s): 4.14 (only)

Issue:: https://issues.redhat.com/browse/TELCODOCS-1671

Link to docs preview: 

- https://70817--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-6

- https://70817--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-5

QE review:
- [x] QE has approved this change.

Additional information:
See https://github.com/openshift/openshift-docs/pull/70249

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
One small suggestion with the bug link.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3334,6 +3334,8 @@ For more information see, xref:../networking/ptp/configuring-ptp.adoc#configurin
 
 * Previously, when you deployed IPv6-only hosts from a dual-stack cluster, an issue prevented the Baseboard Management Controller (BMC) from receiving the correct callback URL. Instead, the BMC received an IPv4 URL. With this update, the issue no longer occurs because the IP version of the URL depends on the IP version of the BMC address.(link:https://issues.redhat.com/browse/OCPBUGS-23903[*OCPBUGS-23903*])
 
+* Previously, on an {sno-caps} that has guaranteed CPUs and where Interrupt Request (IRQ) load balancing is disabled, large latency spikes could occur at container startup. With this update, the issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
```
Comment (JoeAldinger) [inline release_notes/ocp-4-14-release-notes.adoc:3337]: https://issues.redhat.com/browse/OCPBUGS-24281 I think is the bug that is shipped with 4.14.z.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -3365,10 +3367,12 @@ $ oc adm release info 4.14.5 --pullspecs
 ==== Bug fixes
 
 * Previously, when the build capability is not enabled, ConfigObserver controller would fail when trying to sync with build informer. With this release, ConfigObserver starts successfully when build capability is not enabled.
-(link:https://issues.redhat.com/browse/OCPBUGS-23490[*OCPBUGS-23490*])
+(link:https://issues.redhat.com/browse/OCPBUGS-23490[*OCPBUGS-23490*]) (link:https://issues.redhat.com/browse/OCPBUGS-21778[*OCPBUGS-21778*])
```
Comment (JoeAldinger) [inline release_notes/ocp-4-14-release-notes.adoc:3370]:
```suggestion
(link:https://issues.redhat.com/browse/OCPBUGS-23490[*OCPBUGS-23490*]), (link:https://issues.redhat.com/browse/OCPBUGS-21778[*OCPBUGS-21778*])
```

**Comment by kquinn1204** [comment]
@JoeAldinger This is being replaced with https://github.com/openshift/openshift-docs/pull/70817

**Comment by JoeAldinger** [comment]
> @JoeAldinger This is being replaced with #70817

@kquinn1204 I don't follow. Do you mean it is replacing https://github.com/openshift/openshift-docs/pull/70249

**Comment by kquinn1204** [comment]
this is the new one

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (kquinn1204) [reply release_notes/ocp-4-14-release-notes.adoc:3337]: sorry not sure what you want done here @JoeAldinger, this bug OCPBUGS-22901 is described in bugs section so I want to record that as fixed in this z-stream. The JIRA asked me to add this do you want me to add [OCPBUGS-24281](https://issues.redhat.com/browse/OCPBUGS-24281) this in brackets here too ?

Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:3337]: The bug you referenced is marked fixed in 4.15.0. Engr cloned it 24281 for the zstream and that is what the advisory picked up and attached to the advisory referenced in this z-stream. In short, the bug reference needs to be changed or at least add that cloned bug please.

Comment (kquinn1204) [reply release_notes/ocp-4-14-release-notes.adoc:3337]: Thanks for digging that out I will add the cloned bug what is the best practice for that just add link:https://issues.redhat.com/browse/OCPBUGS-24181[*OCPBUGS-24181*])

Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:3337]: Yeah, you can just add the link because the bug already shipped with that zstream.

**Comment by mburke5678** [comment]
@kquinn1204 @JoeAldinger I'm not really following [your conversation above](https://github.com/openshift/openshift-docs/pull/70817#discussion_r1466509742) and how it was resolved. Are we all happy with the current state of the PR for merging?

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:3337]: @mburke5678 that bug needs added.

> [issues.redhat.com/browse/OCPBUGS-24281](https://issues.redhat.com/browse/OCPBUGS-24281) I think is the bug that is shipped with 4.14.z.

**Comment by kquinn1204** [comment]
@mburke5678 1 sec will double check

**Comment by kquinn1204** [comment]
@mburke5678 I added that cloned issue I thought I had should be good tro be merged now. Thanks for picking this up.

**Pull request merged**